File: C++/프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> num_list) {
    int answer = 0;
    string even = "";
    string odd = "";
    
    for (int i=0; i<num_list.size(); i++){
        int num = num_list[i];
        string snum = to_string(num);
        if (num%2 == 0) {
            even += snum;
        } else {
            odd += snum;
        }
    }
    
    answer = stoi(even)+stoi(odd);
    
    
    return answer;
}

File: JavaScript/프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.js
function solution(num_list) {
    let answer = 0;
    let even = '';
    let odd = '';
    
    for (let i=0; i<num_list.length; i++){
        if (num_list[i] % 2 == 0) {
            even += String(num_list[i]);
        } else {
            odd += String(num_list[i]);
        }
    }
    
    answer = Number(even)+Number(odd);
    
    return answer;
}